WindowsServerDocs/identity/ad-fs/operations/ad-fs-single-sign-on-settings.md

```diff
@@ -188,17 +189,21 @@ To Summarize:
   </tr>
 </table>
 
-**Registered Device?** You get a PSSO / Persistent SSO <br>
-**Not Registered Device?** You get a SSO <br>
-**Not Registered Device but KMSI?** You get a PSSO/ Persistent SSO <p>
+**Registered Device?** You get a PSSO / Persistent SSO
+
+**Not Registered Device?** You get a SSO
+
+**Not Registered Device but KMSI?** You get a PSSO/ Persistent SSO
+
 IF:
  - [x] Admin has enabled the KMSI feature [AND]
- - [x] User clicks the KMSI check box on the forms login page
+ - [x] User selects the KMSI check box on the forms login page
 
   
-AD FS issues a new refresh token only if the validity of the newer refresh token is longer than the previous token. The maximum lifetime of a token is is 84 days, but AD FS keeps the token valid on a 14 day sliding window. If the refresh token is valid for 8 hours, which is the regular SSO time, a new refresh token will not be issued.
- 
+AD FS issues a new refresh token only if the validity of the newer refresh token is longer than the previous token. The maximum lifetime of a token is is 84 days, but AD FS keeps the token valid on a 14-day sliding window. If the refresh token is valid for 8 hours, which is the regular SSO time, a new refresh token isn't issued.
+
+**Good to Know:**
+
+Federated users who don't have the **LastPasswordChangeTimestamp** attribute synced are issued session cookies and refresh tokens that have a **Max Age value of 12 hours**.
 
-**Good to Know:** <br>
-Federated users who do not have the **LastPasswordChangeTimestamp** attribute synced are issued session cookies and refresh tokens that have a **Max Age value of 12 hours**.<br>
-This occurs because Azure AD cannot determine when to revoke tokens that are related to an old credential (such as a password that has been changed). Therefore, Azure AD must check more frequently to make sure that the user and associated tokens are still in good standing.
+This occurs because Azure AD can't determine when to revoke tokens that are related to an old credential (such as a password that has been changed). Therefore, Azure AD must check more frequently to make sure that the user and associated tokens are still in good standing.
```

Comment:
Starting with the word this could be confusing when localised. Can we be specific introduce clarity? It is also complex, could it be simplified, this might come from increased clarity.

Reply:
Looks better but Acrolinx is now showing too complex for line 211.

`Max Age value session cookies and refresh tokens are issued because Azure AD can't determine when to revoke tokens that are related to an old credential (such as a password that has been changed).`